sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetReadSupport.scala

```diff
@@ -184,17 +186,6 @@ object ParquetReadSupport extends Logging {
     parquetRequestedSchema
   }
 
-  /**
-   * Overloaded method for backward compatibility with
-   * `caseSensitive` default to `true` and `useFieldId` default to `false`
-   */
-  def clipParquetSchema(
```

Comment:
@sunchao I removed this overloaded method. Let me know if you have any concerns.